documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -59,26 +59,20 @@ metadata:
     createdForProviderType: <provider_type> <2>
 type: Opaque
 stringData:
-  user: <user> <3>
-  password: <password> <4>
-  domainName: <domain_name> <5>
-  projectName: <project_name> <6>
-  regionName: <region name> <7>
-  cacert: | <8>
-    <ca_certificate>
-  url: <api_end_point> <9>
-  thumbprint: <vcenter_fingerprint> <10>
+
+  user: <user> <1>
+  password: <password> <2>
+  insecureSkipVerify: <true/false> <3>
+  cacert: | <4>
+    <engine_ca_certificate>
```

Comment:
we've changed the description of the cacert field - it is also relevant for openstack now, so changing the ca_certification to engine_ca_certificate is incorrect

Reply:
@ahadas please can you clarify?
Thanks

Reply:
the `cacert` field is also used now to set the certificate of an openstack provider so it's incorrect to rename it to `engine_ca_certificate` and change it back to `{rhv-short} only`